src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -72,9 +72,9 @@ protected function configure()
     /**
      * {@inheritdoc}
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $cliOutput)
     {
-        $output = new SymfonyStyle($input, $output);
+        $output = new SymfonyStyle($input, $cliOutput);
```

Comment:
I don't know would such thing be allowed:
75. `protected function execute(InputInterface $input, OutputInterface $output)` - to leave as is
77. `$output = new SymfonyStyle($input, $cliOutput = $output);` - replace with this
88. `return $command->run($input, $cliOutput);`  - replace with this